File: 07_area_of_a_rectangular_room/python/area_of_a_rectangular_room.py
def get_length():
    return int(input("What is the length of the room in feet? "))


def get_width():
    return int(input("What is the width of the room in feet? "))


def calculate_area_in_feet(width, length):
    return width * length


def calculate_area_in_meters(area_in_feet):
    return area_in_feet * 0.09290304


def print_area_in_feet(length, width):
    print(f"You entered dimensions of {length} feet by {width} feet.")


def print_area_in_meters(area_in_feet, area_in_meters):
    print(f"The area is\n{area_in_feet} square feet\n{area_in_meters} square meters")


def main():
    lengthed = get_length()
    widthed = get_width()
    feet_area = calculate_area_in_feet(widthed, lengthed)
    meters_area = calculate_area_in_meters(feet_area)
    print_area_in_feet(lengthed, widthed)
    print_area_in_meters(feet_area, meters_area)


if __name__ == "__main__":
    main()

File: 07_area_of_a_rectangular_room/javascript/area_of_a_rectangular_room.js
const FEET_TO_METERS = 0.09290304;

var length_input = window.prompt("What is the length of the room in feet?");
var width_input = window.prompt("What is the width of the room in feet?");

var length_in_feet = parseInt(length_input);
var width_in_feet = parseInt(width_input);

var area_in_feet = (length_in_feet * width_in_feet);
var area_in_meters = (area_in_feet * FEET_TO_METERS);

console.log(`You entered dimensions of ${length_in_feet} in feet by ${width_in_feet} feet.`)
console.log(`The area is\n${area_in_feet} square feet\n${area_in_meters} square meters.`)
